agent/installer/installer.go

```diff
@@ -52,11 +52,18 @@ func getSupportedRegistry(ob algo.OutputBuilder) registry {
 
 		// BYOH Bundle Repository. Associate bundle with installer
 		linuxDistro := "Ubuntu_20.04.1_x86-64"
-		addBundleInstaller(linuxDistro, "v1.22.3", &algo.Ubuntu20_4K8s1_22{})
+		// If no version specified use the minimum v1.22 supported
+		if k8sVer == "" {
```

Comment:
We don't have to do this. How about removing `k8sVer` argument from the function itself? Any consequences?

Reply:
No consequences at all, actually I introduced this var in this update in order to apply variable matching. I will update it by reverting some changes now and will do the test. Thanks!